sgl-kernel/csrc/cpu/gemm_fp8.cpp

```diff
@@ -89,6 +89,192 @@ inline void unpack_B(
   }
 }
 
+template <typename scalar_t, typename packed_t, bool has_bias, int BLOCK_M, int BLOCK_N>
+struct tinygemm_kernel_nn {
+  static inline void apply(
+      const scalar_t* __restrict__ A, const packed_t* __restrict__ B, scalar_t* __restrict__ C,
+      const float* __restrict__ bias, const float* __restrict__ scale, int K, int lda, int ldb, int ldc, int64_t block_size_K) {
+    TORCH_CHECK(false, "tinygemm_kernel_nn: scalar path not implemented!");
+  }
+};
+
+#if defined(CPU_CAPABILITY_AVX512)
+template <bool has_bias, int BLOCK_M, int BLOCK_N>
+struct tinygemm_kernel_nn<at::BFloat16, at::BFloat16, has_bias, BLOCK_M, BLOCK_N> {
```

Comment:
do we have to keep the bfloat16 tinygemm_kernel_nn in this file? looks like it is not used.

Reply:
It's not needed. Removed in https://github.com/mingfeima/sglang/pull/54/commits/d82f52c97e0ea5a2dda20bf13b619ea3864ac701